policyengine_us/variables/gov/states/ar/tax/income/exemptions/ar_retirement_or_disability_benefits_exemption_indv.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class ar_retirement_or_disability_benefits_exemption_indv(Variable):
+    value_type = float
+    entity = Person
+    label = "Arkansas individual retirement or disability benefits exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=13"
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ar.tax.income.exemptions.retirement_or_disability_benefits
+        head_or_spouse = person("is_tax_unit_head_or_spouse", period)
+        eligible_person = person(
+            "ar_retirement_or_disability_benefits_exemption_eligible_person",
+            period,
+        )
+        pension_income = (
+            person("taxable_pension_income", period) * head_or_spouse
+        )
+        # Filers over a certain age can deduct IRA distributions in addition to pension income
+        ira_distributions = (
+            person("regular_ira_distributions", period) * eligible_person
+        )
+        eligible_pension_income = pension_income + ira_distributions
+        military_retirement_exemption = person(
+            "ar_military_retirement_exemption", period
+        )
+        capped_exemption = max_(
```

Comment:
This produces the right result but we need to assign it to the right variables. To do this, create variables like `_initial` for both the retirement and military exemptions, then create a bool variable indicating that they claim one over the other (based on which is greater), then make the final two variables based on that determination.

Also note that the legal code deviates from the tax form on this front: [§ 26-51-307(f)(2)](https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-3-exemptions-and-reduced-tax-rates/section-26-51-307-retirement-or-disability-benefits-definition) says they can claim a retirement exemption for the difference between $6,000 and the military exemption, but the [tax form](https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=13) only says they cannot take both. Unless I'm misreading this, please file an issue to reconcile these and for now adopt the logic from the tax form.